### PR TITLE
(wmr) - report js-build errors to the client

### PR DESCRIPTION
**Pull request by JoviDeCroock**
image (1)

### CODE REVIEW

File: packages/wmr/src/wmr-middleware.js [hunk 1]
```diff
@@ -316,105 +316,126 @@ export const TRANSFORMS = {
...
-				}
+			writeCacheFile(out, id, code);
 
-				return spec;
+			return code;
+		} catch (e) {
+			if (code && e.loc && e.loc.line) {
+				const splittedCode = code.split('\n');
+
+				e.codeFragment = `
```
Comment (marvinhagemeister) [inline packages/wmr/src/wmr-middleware.js:422]: I wonder if we could reuse the code frame code from here:

https://github.com/preactjs/wmr/blob/57afc7d8395d1bdea37b8ad40fc365b03be96267/packages/wmr/src/lib/output-utils.js#L64

Should be as easy as calling that function and wrapping it in a `kl.stripColors()` which removes the ASCII color notation codes from the string

Comment (marvinhagemeister) [reply packages/wmr/src/wmr-middleware.js:422]: We should probably have a consistent terminology throughout our code base: "code frame" vs "code fragment"

**Review by marvinhagemeister — APPROVED** [review]
Sweet 🙌

**Comment by andyrichardson** [comment]
I like them apples 🚀

**Pull request merged**